### ISSUE TITLE
Quest Completion

### ISSUE THREAD

**Issue by ben-x-dev** [nbcp/firestarter-leaderboard#7]
A player would see a Quest Complete button for his/her own quest
**Completion:** 
- [ ] List of player's assigned quest in his/her profile
1. Clicking complete would show a confirmation modal requiring the user to insert a link to that will go to his/her quest completion proof. 
2. After that the quest will change to pending approval status which would only complete after team lead has validated.

- [ ] email notification for team lead requesting quest approval

**Comment by ben-x-dev** [comment]
This depends on #6

**Comment by ben-x-dev** [comment]
duplicate of #6
